visualization/src/pcl_visualizer.cpp

```diff
@@ -916,7 +916,9 @@ bool
 pcl::visualization::PCLVisualizer::removeText3D (const std::string &id, int viewport)
 {
   // Check to see if the given ID entry exists
-  ShapeActorMap::iterator am_it = shape_actor_map_->find (id);
+  std::string idToCheck(id);
+  if(viewport > 0) idToCheck.append(viewport, '*');
```

Comment:
Missing spaces after `if` and `append`. Also in line above.

Reply:
Hey, do you guys have a formatter like clang-format to follow? So that anyone who contributes will not need to focus too much on the format/coding style?

Reply:
There are style files for Emacs, Uncrustify, and Eclipse (see [here](http://pointclouds.org/documentation/advanced/pcl_style_guide.php#automatic-code-formatting)). There was a [discussion](http://www.pcl-developers.org/Clang-format-style-sheet-td5709356.html) of clang-format style some years ago, but it didn't end up anywhere, unfortunately.